release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -514,6 +514,19 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 
 {product-title} {product-version} now includes CoreDNS version 1.11.3.
 
+[id="ocp-4-17-nw-metallb"]
+==== Changes to MetalLB
+
+With this update, the MetalLB Operator uses `FRR-K8s` as the default backend. 
```

Comment:
I'd just refer to metallb and nto to metallb operator here and after.
The operator is just a mean to deploy metallb.

Reply:
ok. removed mention of "Operator"